assets/css/style.css

```diff
@@ -249,6 +249,7 @@ html[dir=rtl] #srcLanguages {
 
 #srcLangSelectors {
     z-index: 100;
+    width: 90%;
```

Comment:
Yes, here the width of swap button will be constant right? Like I can directly hardcode the value of `swapLangBtn` width in `calc` function? Something like below: ( 35px is the width I am subtracting )

`width: calc(100% - 35px);`